### DISCUSSION TITLE
HmIP-RFUSB statt mit Funkmodul mit Homematic IP Access Point im LAN betreiben

### DISCUSSION THREAD

**Discussion by DonkeeeyKong** [OpenCCU/OpenCCU#2768] [🇩🇪 Fragen&Antworten] [answered]
Hallo, ich würde gerne mit RaspberryMatic loslegen. Da ich allerdings noch nie gelötet habe und auch niemanden kenne, der diesbezüglich helfen könnte, komme ich mit dem HmIP-RFUSB-Bausatz nicht weiter und Fertiggeräte gibt es momentan nirgends. Ich traue mich an das Löten auch nicht so richtig ran, nachdem ich die vielen Threads mit Problemen aufgrund schlechter Verlötung dieses Sticks gefunden habe. 
Ich habe allerdings einen Homematic IP Access Point hier. Verstehe ich [diesen Thread](https://homematic-forum.de/forum/viewtopic.php?f=31&t=79828&start=10#p777192) richtig, dass ich den Stick auch ohne angelötetes Funkmodul nutzen kann, wenn ich einen Access Point im LAN betreibe? Ich habe momentan einige Tisch- und Wandthermostate, Heizungsthermostate sowie Tür- und Fenstersensoren von Homematic IP im Einsatz. Wäre diese Kombination möglich?
Falls nicht: [Dieses Angebot](https://www.smarthome.de/geraete/smarthome-funkstick-fuer-homematic-ip-weiss) der Telekom wirkt für mich sehr verlockend. Ich habe ältere Posts gefunden, in denen davon abgeraten wurde. Ist das noch aktuell, oder hat sich diesbezüglich etwas getan? Hätte ich mit Problemen zu rechnen? Gibt es Erfahrungen mit den Geräten, die ich im Einsatz habe und dem Stick der Telekom? Gäbe es ansonsten anderen Methoden an einen fertig gelöteten Stick zu kommen?
Vielen Dank für alle Antworten!

**Comment by Baxxy13** [comment]
Das du den HmIP-RFUSB in Betracht ziehst bedeutet für mich erstmal du willst keinen SBC (Raspi / Tinker) nutzen. 
Was dann?
Den HmIP-RFUSB ohne angelötetes Funkmodul zu nutzen ist erstmal als "Proof-of-Conzept" anzusehen. Bei meinem kleinen Testsystem funktionierte das (gepaart mit einem HmIP-HAP als LAN-ROUTER) zwar erstaunlich gut, was aber mit richtigen (großen) Systemen passiert ist schwer zu sagen. Außerdem holst du dir einen weiteren möglichen Fehlerpunkt ins System, die LAN-Verbindung.

- wenn du einen SBC (mit GPIO's) nutzen willst dann nutze idealerweise auch das RPI-RF-MOD (Ausnahme Pi4B), die 2 Lötpunkte (beim Bausatz) bekommt jeder hin.
- lass dir den HmIP-RFUSB löten, "Elektronik-Clubs" in Schulen, "Elektronik-Bastlerbuden" oder auch per Kleinanzeige im Homematic-Forum
- Finger weg vom HmIP-RFUSB-TK (Telekom), ja der funktioniert auch für HmIP aber dem fehlt u.a. die Carrier-Sense Anzeige und das Protokoll für HM-Klassik, außerdem ist es ein Telekom-Gerät und bekommt vermutlich nie eine offizielle eQ-3 Firmware

**Reply by DonkeeeyKong** [reply]
Hallo, vielen Dank für deine Antwort! Ich habe einen kleinen x86-Homeserver, den ich dafür benutzen möchte. Ich habe mir jetzt den RFUSB-Bausatz bestellt. Vielleicht funktioniert's ja gut ohne die Antenne, ansonsten muss ich wohl versuchen, das irgendwo löten zu lassen. Es scheint ja aber nichts dagegen zu sprechen, es erstmal über den HAP zu versuchen.

**Comment by DonkeeeyKong** [comment, marked as answer]
So: Kurze Rückmeldung. Vielleicht interessiert's ja jemanden:

Ich habe jetzt den HmIP-RFUSB erhalten und erstmal ohne Funkmodul (also ohne zu löten) angeschlossen und an meine VM, in der RaspberryMatic läuft, weitergereicht. Dann habe ich den Access Point und danach alle Thermostate, Sensoren etc. auf den Werkszustand zurückgesetzt. Den Access Point konnte ich problemlos anlernen und danach auch alle Geräte vollkommen problemlos anlernen und nutzen. Das Ganze über custom_homematic bei Home Assistant einrichten war ebenfalls kein Problem.

Zusammengefasst: Bisher sehe ich keinen Grund, das Funkmodul anzulöten. Ich habe eher Angst damit etwas zu verschlimmbessern. Das System läuft über den HAP bisher absolut tadellos!

**Reply by dreamworks** [reply]
Hi,

Access Point hab ich auch so integrieren können, aber ich schaffe es nicht die HmIP-eTRV-B-2 and die RaspberryMatic anzulernen.. In der CCU-UI bekomme ich über IP Geräte neu anlernen keine Nachrichten im Posteingang :-/

**Reply by DonkeeeyKong** [reply]
Hm. Den habe ich nicht, sondern den HmIP-eTRV-2. Das war allerdings wie beschrieben kein Problem und das System läuft auch ein halbes Jahr später immer noch vollkommen problemlos und wie gewünscht.

**Reply by flo89** [reply]
hi!
@DonkeeeyKong 
Ich habe dasselbe Setup, ich bekomme aber aus irgendeinem Grund nichts via dem AccessPoint angelernt.
Bei dir funktioniert das?

**Reply by DonkeeeyKong** [reply]
> hi! @DonkeeeyKong Ich habe dasselbe Setup, ich bekomme aber aus irgendeinem Grund nichts via dem AccessPoint angelernt. Bei dir funktioniert das?

Hi. Was meinst du mit „via dem Access Point angelernt“? Ich habe den Access Point auf den Werkszustand zurückgesetzt und dann den angelernt und in RaspberryMatic hinzugefügt (LED des Access Points leuchtet jetzt türkis. Nicht mehr blau!). Danach konnte ich problemlos alle Geräte anlernen und in RaspberryMatic hinzufügen.
Das ganze Setup läuft seitdem unverändert und problemlos ohne das Funkmodul wie oben beschrieben.

**Reply by flo89** [reply]
> Hi. Was meinst du mit „via dem Access Point angelernt“? Ich habe den Access Point auf den Werkszustand zurückgesetzt und dann den angelernt und in RaspberryMatic hinzugefügt (LED des Access Points leuchtet jetzt türkis. Nicht mehr blau!). Danach konnte ich problemlos alle Geräte anlernen und in RaspberryMatic hinzufügen. Das ganze Setup läuft seitdem unverändert und problemlos ohne das Funkmodul wie oben beschrieben.

Mit "via Access Point" meinen ich, dass der Access Point die ganze Kommunikation während dem Anlernen übernommen hatt, da dein Stick ja mangels Sende Modul nichts senden und empfangen kann.

Ich kann bei meinem Setup Geräte die schon vorhanden sind aus dem Backup steuern aber keine neuen Anlernen.

Ich hab im Log dan folgende Meldung was mich vermuten ließ das der AP keine Geräte anlernen darf:

Don't use Router AFA823, not supported for Inclusion Accept

**Reply by DonkeeeyKong** [reply]
> > Hi. Was meinst du mit „via dem Access Point angelernt“? Ich habe den Access Point auf den Werkszustand zurückgesetzt und dann den angelernt und in RaspberryMatic hinzugefügt (LED des Access Points leuchtet jetzt türkis. Nicht mehr blau!). Danach konnte ich problemlos alle Geräte anlernen und in RaspberryMatic hinzufügen. Das ganze Setup läuft seitdem unverändert und problemlos ohne das Funkmodul wie oben beschrieben.
> 
> Mit "via Access Point" meinen ich, dass der Access Point die ganze Kommunikation während dem Anlernen übernommen hatt, da dein Stick ja mangels Sende Modul nichts senden und empfangen kann.
> 
> Ich kann bei meinem Setup Geräte die schon vorhanden sind aus dem Backup steuern aber keine neuen Anlernen.
> 
> Ich hab im Log dan folgende Meldung was mich vermuten ließ das der AP keine Geräte anlernen darf:
> 
> Don't use Router AFA823, not supported for Inclusion Accept

Hm. Wie hast du denn die Geräte, die schon vorhanden sind, hinzugefügt?

**Reply by flo89** [reply]
Einfach ein Backup eingespielt

**Comment by DonkeeeyKong** [comment]
Nach über einem Monat sind keinerlei Probleme aufgetreten. Fazit: Es funktioniert!
